animation_nodes/nodes/mesh/insert_uv_map.py

```diff
@@ -0,0 +1,25 @@
+import bpy
+from ... data_structures import VirtualVector2DList
+from ... base_types import AnimationNode, VectorizedSocket
+
+class InsertUVMapNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_InsertUVMapNode"
+    bl_label = "Insert UV Map"
+    errorHandlingType = "EXCEPTION"
+
+    def create(self):
+        self.newInput("Mesh", "Mesh", "mesh", dataIsModified = True)
+        self.newInput("Text", "Name", "uvMapName", value = "AN-UV Map")
+        self.newInput("Vector 2D List", "Vectors", "vectors")
```

Comment:
This should be a vectorized socket.

Reply:
But we have to input the vector list, not a vector.

Reply:
Just like the vertex color nodes, since we use virtual lists, a single input will materialize to a list.

Reply:
Okay.